src/m365/spo/commands/site/site-commsite-enable.spec.ts

```diff
@@ -76,75 +72,17 @@ describe(commands.SITE_COMMSITE_ENABLE, () => {
   });
 
   it('enables communication site features on the specified site', async () => {
```

Comment:
In fact, this is exactly the same test as the one below, only without debug option. Let's remove this one and just keep the debug test.